view/frontend/templates/express/checkout-button.phtml

```diff
@@ -44,7 +45,7 @@ if ($customer->getDefaultShippingAddress()) {
         <?= $formSelector ? 'form-selector="' . $escaper->escapeHtmlAttr($formSelector) . '"' : '' ?>
          id="<?= $escaper->escapeHtmlAttr($buttonId) ?>" initial-email="<?= $escaper->escapeHtmlAttr($customer->getEmail()) ?>" initial-first-name="<?= $escaper->escapeHtmlAttr($customer->getFirstname()) ?>"
          initial-last-name="<?= $escaper->escapeHtmlAttr($customer->getLastname()) ?>" initial-phone-number="<?= $escaper->escapeHtmlAttr($phone_number ?? '') ?>"
-         initial-address="<?= $escaper->escapeHtmlAttr($formatedAddress ?? '') ?>">
+         initial-address="<?= $escaper->escapeHtmlAttr($formatedAddress ?? '') ?>" post-code-optional-countries="<?= $escaper->escapeHtmlAttr($postcodeOptionalCountries) ?>">
```

Comment:
I haven't looked closely here before, why do we keep add attributes to the containing div. the only two attributes needed are `class` and `trigger-context`